core/src/banking_stage/transaction_scheduler/transaction_state_container.rs

```diff
@@ -44,6 +44,7 @@ pub(crate) struct TransactionStateContainer<Tx: TransactionWithMeta> {
     capacity: usize,
     priority_queue: MinMaxHeap<TransactionPriorityId>,
     id_to_transaction_state: Slab<TransactionState<Tx>>,
+    held_transactions: Vec<TransactionPriorityId>,
```

Comment:
Just to confirm that the addition of `held_transactions` won't meaningfully change memory usage, cause retryable transactions are either in priority_queue or in help_transactions?